pyscf/dft/rks.py

```diff
@@ -339,6 +382,11 @@ def __init__(self, xc='LDA,VWN'):
 # don't modify the following attributes, they are not input options
         self._numint = numint.NumInt()
 
+        # disable nlc for certain xc such as wb97m-d3bj
+        if with_nlc is not None:
```

Comment:
if one sets `nlc`, or `xc` after creating RKS object, the `_numint.nlc_coeff` and `self._numint.is_nlc` can never parse the xc correctly. Are there any other ways to control this?

Reply:
`nlc` is possibly disabled only when creating RSK object with certain functionals such as wb97m-d3bj. After the creation, setting nlc or xc does not change `self._numint.is_nlc`. The functionality of setting nlc or xc after creating RKS object is the same as before.

And one still can use `self._numint.libxc.is_nlc` and `self._numint.libxc.nlc_coeff` to parse xc code if `self._numint.is_nlc` is customized.

Reply:
Yes, it is better to customize the parser in those two functions.
https://github.com/wxj6000/pyscf/blob/dft_wrapper/pyscf/dft/numint.py#L2647
and 
https://github.com/wxj6000/pyscf/blob/dft_wrapper/pyscf/dft/numint.py#L2644
I will work on it later.